netmanager-app/core/apis/MapData.ts

```
@@ -0,0 +1,59 @@
+                {
+                        console.log("Error Token Is Missing")
+                        return null;
+                }
+                
+        }
+        catch (error) {
+                console.error('Error fetching GeoJSON:', error);
+        }
+}
+
+export  const FetchSuggestions=async(value:string,access_token:string,sessionToken:string,latitude?: number, longitude?: number):Promise<any[] | void>=>{
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)